mysql/gtid.go

```diff
@@ -15,6 +18,8 @@ type GTIDSet interface {
 	Update(GTIDStr string) error
 
 	Clone() GTIDSet
+
+	AddGTID(uuid.UUID, int64, uint32, uint32, uint64) error
```

Comment:
this function signature is very unfriendly. I prefer we don't add it.

We can use type assert to get MariadbGTIDSet and MysqlGTIDSet and call its AddGTID